templates/repo/blame.tmpl

```diff
@@ -16,11 +16,13 @@
 				{{end}}
 				<a class="ui tiny button" href="{{.RepoLink}}/src/{{.BranchNameSubURL}}/{{.TreePath | PathEscapeSegments}}">{{.i18n.Tr "repo.normal_view"}}</a>
 				<a class="ui tiny button" href="{{.RepoLink}}/commits/{{.BranchNameSubURL}}/{{.TreePath | PathEscapeSegments}}">{{.i18n.Tr "repo.file_history"}}</a>
+				<a class="ui tiny button unescape-button">{{.i18n.Tr "repo.unescape_control_characters"}}</a>
+				<a class="ui tiny button escape-button" style="display: none;">{{.i18n.Tr "repo.escape_control_characters"}}</a>
```

Comment:
Could there be a setting to have code not escaped by default? This would help if you're regularly reviewing bidirectional text and don't consider yourself at risk of this vulnerability.

Could there also be an option to disable showing the warning symbols?

Reply:
Just change the template.

Reply:
I mean a user-specific option rather than an instance option. I use a few Gitea instances but do not administer them, so I cannot change the template.